src/modals/UnlockChunks/Overview.tsx

```diff
@@ -7,26 +7,29 @@ import { ButtonSubmit } from '@rossbulat/polkadot-dashboard-ui';
 import BigNumber from 'bignumber.js';
 import { useApi } from 'contexts/Api';
 import { useNetworkMetrics } from 'contexts/Network';
+import { Countdown } from 'library/Countdown';
 import { useErasToTimeLeft } from 'library/Hooks/useErasToTimeLeft';
+import { useTimeLeft } from 'library/Hooks/useTimeLeft';
 import { fromNow, timeleftAsString } from 'library/Hooks/useTimeLeft/utils';
 import { useUnstaking } from 'library/Hooks/useUnstaking';
 import { StatsWrapper, StatWrapper } from 'library/Modal/Wrappers';
-import { forwardRef } from 'react';
+import { forwardRef, useEffect } from 'react';
 import { useTranslation } from 'react-i18next';
 import { planckToUnit } from 'Utils';
 import { NotesWrapper, Separator } from '../Wrappers';
 import { ChunkWrapper, ContentWrapper } from './Wrappers';
 
 export const Overview = forwardRef(
   ({ unlocking, bondFor, setSection, setUnlock, setTask }: any, ref: any) => {
-    const { network, consts } = useApi();
+    const { network, consts, status: connectionStatus } = useApi();
```

Comment:
```suggestion
    const { network, consts, apiStatus } = useApi();
```

`status` almost always requires an alias for this. Can you rename `status` to `apiStatus` and also update `connectionStatus` in `contexts/API` to `apiStatus` also.

Reply:
Done, just leave  [type ConnectionStatus](https://github.com/TingALin/polkadot-staking-dashboard/blob/ting-unlock-era/src/contexts/Api/types.ts#L9 )  unchanged as i think this's more intuitive

Reply:
Why change it for this one file?